### PR TITLE
support capsule v1beta1

### LINKED ISSUE #122: Support v1beta1 tenants API
As capsule is going to be updated to `v1beta1` version, we need to support this changes in capsule-proxy too.

### PR DESCRIPTION
**Pull request by MaxFedotov**
closes #122

### CODE REVIEW

File: e2e/run.bash [hunk 1]
```diff
@@ -10,4 +10,5 @@ echo ">>> Waiting for capsule pod to be ready for accepting requests"
 kubectl --namespace capsule-system wait --for=condition=ready --timeout=320s pod -l app.kubernetes.io/instance=capsule
 
 echo ">>> Starting test suite"
+#bats -t "$(git rev-parse --show-toplevel)"/e2e/tests/*
```
Comment (prometherion) [inline e2e/run.bash:13]: Maybe a typo here?

File: e2e/libs/serviceaccount_utils.bash [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  name: sa
+current-context: sa
+kind: Config
+preferences: {}
+users:
+- name: ${name}
+  user:
+    token: ${TOKEN}
+EOF
+}
```
Comment (prometherion) [inline e2e/libs/serviceaccount_utils.bash:40]: Missing new line at the end of the file

File: e2e/libs/storageclass_utils.bash [hunk 3]
```diff
@@ -14,8 +14,6 @@ provisioner: capsule.clastix.io/${name}
 reclaimPolicy: Delete
 volumeBindingMode: WaitForFirstConsumer
 EOF
-
-  sleep 1
```
Comment (prometherion) [inline e2e/libs/storageclass_utils.bash:18]: Is it safe removing the sleep?

I recall something odd without it.

Comment (prometherion) [reply e2e/libs/storageclass_utils.bash:18]: Hum, but tests are ok, so I think it's safe to remove it.

File: e2e/run.bash (hunk 1 above)
Comment (MaxFedotov) [reply e2e/run.bash]: yep, forgot to remove line

File: e2e/libs/serviceaccount_utils.bash (hunk 2 above)
Comment (MaxFedotov) [reply e2e/libs/serviceaccount_utils.bash]: fixed

**Review by prometherion — APPROVED** [review]

**Pull request merged**